HotelBooking.Model/Entities/ImgUrl.cs

```diff
@@ -0,0 +1,16 @@
+﻿using HotelBooking.Common.Base;
+using System.ComponentModel.DataAnnotations;
+
+namespace HotelBooking.Model.entity
+{
+    public class ImgUrl : BaseEntity
```

Comment:
Rename this class to Image,
For this entity, you define it as a model then hotel will use this entity to refer
Please ask me for this one

Reply:
Image:
Id
Url
Description
...

Reply:
instead of only image, we can define it as file